plugins/toolbox/src/components/Converters/SLACalculator.tsx

```diff
@@ -0,0 +1,122 @@
+import { useStyles } from '../../utils/hooks';
+import React from 'react';
+import {
+  Divider,
+  FormControl,
+  Grid,
+  TextField,
+  Typography,
+} from '@material-ui/core';
+import { PasteFromClipboardButton } from '../Buttons/PasteFromClipboardButton';
+import { ClearValueButton } from '../Buttons/ClearValueButton';
+import { CopyToClipboardButton } from '../Buttons/CopyToClipboardButton';
+
+export const SLACalculator = () => {
+  const styles = useStyles();
+  const [input, setInput] = React.useState('');
+  const [output, setOutput] = React.useState({
+    daily: '',
+    weekly: '',
+    monthly: '',
+    quarterly: '',
+    yearly: '',
+  });
+
+  const convertTime = (value: number) => {
+    let minutes = Math.floor(value / 60);
+    const seconds = Math.floor(value % 60);
+    if (minutes >= 60) {
+      const hours = Math.floor(minutes / 60);
+      minutes = minutes % 60;
+      return `${hours}h ${minutes}m ${seconds}s`;
+    }
+
+    return `${minutes}m ${seconds}s`;
+  };
+
+  const handleChange = (value: string) => {
+    if (value.length === 0) {
+      setInput('');
+      setOutput({
+        daily: '',
+        weekly: '',
+        monthly: '',
+        quarterly: '',
+        yearly: '',
+      });
+      return;
+    }
+
+    const base = parseFloat(value);
+    if (isNaN(base)) {
```

Comment:
I think there should be some error for the user in case invalid input is given

Reply:
Added error field that will appear based on user input.